src/journal/fuzz-journald-audit.c

```diff
@@ -7,6 +7,9 @@
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         Server s;
 
+        if (size > 65536)
```

Comment:
I think the journald fuzzers should reject buffers that are a bit larger than the underlying functions are supposed to handle. Looking at server_process_audit_message it seems it receives messages via a socket so this limit should be a bit larger that what can be received via that socket.

The fuzzers fuzzing various argv functions should reject buffers that are much larger than 65536 bytes to be able to cover all the corner cases.

Anyway, I'm not sure I can pick sensible defaults in a reasonable amount of time :-)